schema/deploy/shipping/views.sql

```diff
@@ -3134,9 +3138,14 @@ create or replace view shipping.__uw_priority_queue_v1 as (
             jsonb_extract_path_text (encounter.details, '_provenance', 'redcap', 'event_name' ) as redcap_event_name,
             jsonb_extract_path_text (encounter.details, '_provenance', 'redcap', 'repeat_instance' ) as redcap_repeat_instance,
             tier,
-            current_prev_pos,
             latest_invite_date,
-            latest_collection_date
+            latest_collection_date,
+            coalesce(invitation_count, 0) as invitation_count,
+            uw_housing_group,
+            on_campus_2x_week,
+            latest_positive_hcov19_collection_date,
+            latest_prior_test_positive_date,
```

Comment:
Do we need to include `prior_test_positive_base` field here in case a participant has never done a daily attestation?

Reply:
I've brought in the `prior_test_positive_date_base` date from the enrollment survey and used that at the bottom for uber filtering. We might as well use it since we've got it!

Reply:
Would it make sense to coalesce the two fields instead of adding an additional field/filter?

Reply:
The individual could have a different value for `prior_test_positive_date_base` and `latest_prior_test_positive_date`. For example, they give '2020-10-01' on their enrollment. In their daily attestation once they give '2020-12-15'.
We wouldn't want a coalesce here. We'd want the max value and compare that against 90 days.
We could max() all three values, assuming that max works like this ... 
In Postgres this does not work:
select max('2020-01-01'::date, '2020-02-01'::date, '1980-02-02'::date)
Ah, it's `greatest`  https://www.postgresql.org/docs/9.5/functions-conditional.html#FUNCTIONS-GREATEST-LEAST :+1: 

"The GREATEST and LEAST functions select the largest or smallest value from a list of any number of expressions. The expressions must all be convertible to a common data type, which will be the type of the result (see Section 10.5 for details). NULL values in the list are ignored. The result will be NULL only if all the expressions evaluate to NULL."

This does work in Postgres:
select greatest(null, '2020-02-01'::date, '1980-02-02'::date)

This ends up looking complicated:
where
 ( greatest(latest_positive_hcov19_collection_date, latest_prior_test_positive_date, prior_test_positive_date_base) 
    is null or greatest(latest_positive_hcov19_collection_date, latest_prior_test_positive_date, prior_test_positive_date_base) < current_date - interval '90 days')
I could compute the `greatest` earlier and then use that, but to keep things simpler I'm going to leave this as it is with the 3 different date evaluation clauses.

Reply:
Ah, so you can use the `greatest` function within the `uw_enrollments` CTE to consolidate all three dates into one date field.

Reply:
Right. I could do that and not carry down the individual dates. That ends up with cleaner looking code at the bottom, but I lose the flexibility of having the separate columns. But I could keep all 3 columns and do the `greatest` and carry them all down. 
The end result is the same, so I don't see a huge reason to do that.